legend-engine-xt-serviceStore-grammar/src/main/java/org/finos/legend/engine/language/pure/grammar/from/ServiceStoreParseTreeWalker.java

```diff
@@ -368,16 +364,10 @@ private TypeReference visitTypeReference(ServiceStoreParserGrammar.TypeReference
         return typeReference;
     }
 
-    private SecurityScheme visitSecurityScheme(ServiceStoreParserGrammar.IdentifierContext securitySchemeCtx)
+    private String visitSecurityScheme(ServiceStoreParserGrammar.IdentifierContext securitySchemeCtx)
```

Comment:
lets not have this change, this would be backward incompatible. No changes in grammar with this PR as any such change impacts user